examples/convolution/stb

```diff
@@ -0,0 +1 @@
+../../vendor/stb
```

Comment:
I don't think this works on Windows, but we can worry about that later.

Reply:
There's symlink support for windows-git now, but I think it requires special configuration. We should probably just place a copy of `stb` in both example folders, git will de-duplicate them anyway.